Solutions/BitSight/Parsers/BitSightCompanyDetails.txt

```diff
@@ -0,0 +1,114 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias (e.g. BitSightCompanyDetails).
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. BitSightCompanyDetails | take 10).
+// References : 
+// Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// Tech Community Blog on KQL Functions : https://techcommunity.microsoft.com/t5/Azure-Sentinel/Using-KQL-functions-to-speed-up-analysis-in-Azure-Sentinel/ba-p/712381
+//
+let CompanyDetails_view  = view () { 
+    Company_details_CL
+    | extend 
+                EventVendor ="CompanyDetailsBitSight",
+                EventProduct ="CompanyDetailsBitSight",
+                PrimaryCompanyGUID = column_ifexists('primary_company_guid_g', ''),
+                PrimaryCompanyName = column_ifexists('primary_company_name_s', ''),
```

Comment:
The sample data contained company_name only and thus will not get mapped.

Reply:
@devikamehra I have updated correct sample data.